flocker/docs/version_extensions.py

```diff
@@ -111,7 +112,31 @@ def run(self):
         return CodeBlock.run(self)
 
 
+# Due to the dash in the name, the sphinx-prompt module is unloadable
+# using a normal import - use the importlib machinery instead.
+sphinx_prompt = importlib.import_module('sphinx-prompt')
+
+
+class VersionPrompt(sphinx_prompt.PromptDirective):
+    """
+    Similar to PromptDirective but replaces a placeholder with the
+    latest installable version of Flocker.
+
+    Usage example:
+
+    .. version-prompt:: bash $
+
+       $ brew install flocker-|latest-installable|
+    """
+    def run(self):
```

Comment:
Ideally, if we decide to convert all code-blocks to prompts, to address FLOC-1388, the code-block version could be deleted. Hence, I think refactoring to share code is not useful at this stage.